pkg/commands/string-to-args.go

```diff
@@ -0,0 +1,114 @@
+// ToArgv is copied from github.com/mgutz/str
+
+package commands
+
+import "runtime"
+
+// ToArgv converts string s into an argv for exec.
+func ToArgv(s string) []string {
```

Comment:
I don't fully understand what shortcoming of the previous implementation this method attempts to resolve

Reply:
This was the only function in a big package so i thought why not include it directly inside a file. If that's not a problem i can do it the old way again

Reply:
I have switched it back to the old way